app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -108,9 +129,7 @@ export default async function Page({ params }: { params: ParamsType }) {
     notFound();
   }
 
-  const bridgeOptions = { resolveRelations };
-  let slug = params.slug ? params.slug.join('/') : '';
   return (
-    <StoryblokStory story={data.story} bridgeOptions={bridgeOptions} slug={slug} />
```

Comment:
Why did you have the slug here previously?

Reply:
Oh I might keep this for now. I think I might end up needing this for social sharing. I added it back then thinking that it might come in handy at one point.

Reply:
I put the slug back.

Reply:
Thanks 😆 If I have time for maintenance at the end I'll see whether we actually need it.